docker/neurodocker_gen_base.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
+
+docker run --rm repronim/neurodocker:latest \
+  generate docker \
+    --base-image ubuntu:22.04 \
```

Comment:
Not happy with using a ubuntu image, kinda bloated. But it's the only one that works out-of-the-box.